### PR TITLE
Update gpu_advanced.md

### PR DESCRIPTION
**Pull request by Saduf2019**
Updating "thThis" "This" as per #53775

### CODE REVIEW

**Review by gbaned — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by mihaimaruseac** [comment]
In the future, can you try to fix multiple typos at once, please? It doesn't make sense to run all of these hours of CI for just 1-2 letters changes.

**Comment by gbaned** [comment]
@Saduf2019  Can you please check @mihaimaruseac's comments and resolve conflicts?. Thanks!

File: tensorflow/lite/g3doc/performance/gpu_advanced.md [hunk 1]
```diff
@@ -452,7 +452,7 @@ internal dequantized buffer.
 
 Using serialization of GPU kernel code and model data from previous
 initializations can reduce latency of GPU delegate's initialization up to 90%.
-ThThis improvement is is achieved by exchanging disk space for time savings. You
+This improvement is is achieved by exchanging disk space for time savings. You
```
Comment (mihaimaruseac) [inline tensorflow/lite/g3doc/performance/gpu_advanced.md:455]:
```suggestion
This improvement is achieved by exchanging disk space for time savings. You
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by mihaimaruseac** [comment]
6eb4c6bc38c81b54b93b063bee41dbe530b4174f

**Pull request closed without merging**